pages/stable-version.html

```diff
@@ -90,25 +90,25 @@ <h1>Stable version 2023.04</h1>
         <div class="row content-padding subpage-content">
 
             <h4><a href="#releasenotes">Release notes</a></h4>
-            <p><strong>WARNING:</strong> Remember to review the release notes! They contain valueble information and, depending on your current version, required procedures to update to the most recent one! Make sure you read the release notes for all the versions between the one you are using now, and 2023.04!</strong></p>
+            <p><strong>WARNING:</strong> Remember to review the release notes! They contain valuable information and, depending on your current version, required procedures to update to the most recent one! Make sure you read the release notes for all the versions between the one you are using now, and 2023.09!</strong></p>
```

Comment:
I'd comment this part and replace it with another message:
```suggestion
            <!--<p><strong>WARNING:</strong> Remember to review the release notes! They contain valuable information and, depending on your current version, required procedures to update to the most recent one! Make sure you read the release notes for all the versions between the one you are using now, and 2023.09!</strong></p>-->
            <p><strong>WARNING:</strong> Remember to review the release notes! They contain valuable information and, for the update to 2023.09, a major upgrade with a special procedure is required, as the base operating system is now changing to openSUSE Leap 15.5 for the Uyuni Server and the Proxies!</strong></p>
```

Then for 2023.10, you remove this second line, and uncomment the first one :-)